local-setup/telemetry/sidecar.sh

```
@@ -0,0 +1,137 @@
+VISITOR_ID=$(printf '%s%s' \
+  "$(cat /etc/hostname 2>/dev/null || hostname)" \
+  "$(cat /sys/class/net/eth0/address 2>/dev/null || echo none)" \
+  | sha256sum | cut -c1-16)
+
+# ── Telemetry sender (fire-and-forget, backgrounded) ─────────────
+# Uses a real user-agent so Matomo doesn't filter it as bot traffic.
+UA="DIGIT-LocalSetup/1.0"
+
+send_event() {
+  [ "${TELEMETRY:-true}" = "false" ] && return 0
+  local category="$1" action="$2" name="$3"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)